### PR TITLE
Remove phantomjs and default to chrome-headless.

### PR DESCRIPTION
**Pull request by electroniceagle**

### CODE REVIEW

**Comment by electroniceagle** [comment]
[Deprecate phantomjs support.](https://trello.com/c/lfW78kiH/2151-deprecate-phantomjs-support)

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,24 @@
-sudo: false
+sudo: required
```
Comment (cworth-gh) [inline .travis.yml:1]: This hunk isn't described in the commit message. Separate to a new commit? (Similarly, below several times---I'll make a shorter comment in each case.)

File: .travis.yml [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
 language: python
-addons:
-  chrome: beta
-  firefox: latest
 python:
   - "2.7"
+addons:
+  chrome: stable
```
Comment (cworth-gh) [inline .travis.yml:6]: Separate commit for changing chrome version from "beta" to "stable"?

File: .travis.yml [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
+  firefox: latest
 env:
   matrix:
     - TEST_BROWSER=firefox SELENIUM_WINDOW_SIZE=1024x768
-    - TEST_BROWSER=chrome SELENIUM_WINDOW_SIZE=1024x768
     - TEST_BROWSER=firefox SELENIUM_WINDOW_SIZE=640x480
+    - TEST_BROWSER=chrome SELENIUM_WINDOW_SIZE=1024x768
```
Comment (cworth-gh) [inline .travis.yml]: Separate commit for reordering the matrix to put the browsers next to each other? (Perhaps even in the earlier commit in this series that changes this matrix a lot.)

File: .travis.yml [hunk 4]
```diff
@@ -1,23 +1,24 @@
...
     - TEST_BROWSER=chrome SELENIUM_WINDOW_SIZE=640x480
     - TEST_BROWSER=firefox-headless SELENIUM_WINDOW_SIZE=1024x768
-    - TEST_BROWSER=chrome-headless SELENIUM_WINDOW_SIZE=1024x768
     - TEST_BROWSER=firefox-headless SELENIUM_WINDOW_SIZE=640x480
+    - TEST_BROWSER=chrome-headless SELENIUM_WINDOW_SIZE=1024x768
     - TEST_BROWSER=chrome-headless SELENIUM_WINDOW_SIZE=640x480
   global:
     - DISPLAY=:99.0
 install:
+  - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then travis_retry pip install "Django<2.0"; fi
```
Comment (cworth-gh) [inline .travis.yml]: Ah! This is the piece that _is_ described in the commit message. Nothing to separate here. ;-)

File: selenium_testcase/testcases/headless.py [hunk 5]
```diff
@@ -9,6 +9,7 @@ def Chrome():
     """ Launch chrome selenium webdriver with headless option. """
     options = webdriver.ChromeOptions()
     options.add_argument('--headless')
+    options.add_argument('--no-sandbox')
```
Comment (cworth-gh) [inline selenium_testcase/testcases/headless.py]: What's the --no-sandbox option? Does this belong in the earlier commit that adds --headless?

**Comment by electroniceagle** [comment]
I'll refactor.  The downside of testing travis-only failures...

**Pull request merged**